cronjob.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+ name: hello-cronjob
+ namespace: mateapp
+spec:
+ schedule: "*/4 * * * *"
+ concurrencyPolicy: Allow
```

Comment:
The `concurrencyPolicy` is set to 'Allow', which means multiple instances of the job can run concurrently. If this is not the desired behavior, consider changing it to 'Forbid' or 'Replace' to prevent overlapping runs.